pkg/rtc/mediatrack.go

```diff
@@ -212,10 +212,9 @@ func (t *MediaTrack) AddReceiver(receiver *webrtc.RTPReceiver, track *webrtc.Tra
 		}
 
 		t.MediaTrackReceiver.SetupReceiver(wr)
-		t.startMaxQualityTimer()
```

Comment:
Result of lot of refactoring and embedding :-(. I noticed `stopMaxQualityTimer` a couple of days back and fixed that. Just now noticed this.